Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-theme.tsx

```diff
@@ -0,0 +1,48 @@
+import { useDispatch, useSelector } from 'react-redux';
+
+import { toggleTheme } from '../redux/features/themeSlice';
+
+interface ITheme {
+    [key: string]: {
+        textColor: string;
+        baseColor100: string;
+        baseColor200: string;
+        baseColor300: string;
+        baseColor400: string;
+        baseColor500: string;
+    };
+}
+const useTheme = () => {
+    const dispatch = useDispatch();
+    const { mode } = useSelector((state: any) => state.theme);
+
+    const toggleSelectedTheme = () => {
+        dispatch(toggleTheme());
+    };
+
+    const themeColors: ITheme = {
+        // TBD
+        light: {
+            textColor: 'black',
+            baseColor100: '',
+            baseColor200: '',
+            baseColor300: '',
+            baseColor400: '',
+            baseColor500: '',
+        },
+        dark: {
+            textColor: 'white',
+            baseColor100: '#687487',
+            baseColor200: '#374151',
+            baseColor300: '#2C2F37',
+            baseColor400: '#212328',
+            baseColor500: '#1C1C17',
```

Comment:
Why not put these also in a css file with variables?

Reply:
because it will be variable pointing to variable, which finally points to value, so changing it would require going thtough mupltiple files, instead of just using the config object itself as a theme itself

Reply:
It would require just one change, to update a value, or I don't understand what you mean. Anyways I guess it's ok to stay like this, if these colors will be referenced only from the theme and by no other way..